src/components/ui/badge.jsx

```
@@ -1,27 +1,28 @@
-import { forwardRef } from "react"
-import { cn } from "@/lib/utils"
+import React, { forwardRef } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)